### PR TITLE
doc: edits while Amy away

### PR DESCRIPTION
**Pull request by dbkinder**
- update getting_started_hello.rst with numbered-step directive
- break long lines into shorter ones (80 chars)
- add changes indicated in GSG review of word doc, fix instruction steps, missing steps
- clarity and further explanation edits

Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

File: doc/getting-started/getting_started_hello.rst [hunk 1]
```diff
@@ -3,19 +3,27 @@
 Getting Started Guide
 #####################
 
-This guide will help you get started with ACRN. We’ll show how to prepare a build environment on your development computer. Then we’ll walk through the steps to set up ACRN’s predefined shared scenario on a target system, with the goal of running ACRN hypervisor, Service VM, and one Ubuntu guest VM, as illustrated in this figure:
+This guide will help you get started with ACRN. We'll show how to prepare a
+build environment on your development computer. Then we'll walk through the
+steps to set up an ACRN predefined **shared** scenario on a target system and
+run the ACRN hypervisor, Service VM, and one Ubuntu User VM, as
```
[the remark this answers is not in the document]
Comment (amyreye) [reply doc/getting-started/getting_started_hello.rst:9]: @dbkinder I'll update this image. Should I do it here in your branch?

[the remark this answers is not in the document]
Comment (dbkinder) [reply doc/getting-started/getting_started_hello.rst:9]: You can submit your own PR to do that, thanks.

[the remark this answers is not in the document]
Comment (amyreye) [reply doc/getting-started/getting_started_hello.rst:9]: Will do

File: doc/getting-started/getting_started_hello.rst [hunk 2]
```diff
@@ -3,19 +3,27 @@
 Getting Started Guide
 #####################
 
-This guide will help you get started with ACRN. We’ll show how to prepare a build environment on your development computer. Then we’ll walk through the steps to set up ACRN’s predefined shared scenario on a target system, with the goal of running ACRN hypervisor, Service VM, and one Ubuntu guest VM, as illustrated in this figure:
+This guide will help you get started with ACRN. We'll show how to prepare a
+build environment on your development computer. Then we'll walk through the
+steps to set up an ACRN predefined **shared** scenario on a target system and
```
[the remark this answers is not in the document]
Comment (amyreye) [reply doc/getting-started/getting_started_hello.rst:8]: @dbkinder Cory suggests using future scenario names for the study to gather feedback on them. But it sounds like we will need to revert them to the current names for the 2.6 release.

**Pull request merged**